src/components/TeamForm/TeamForm.js

```diff
@@ -68,6 +97,30 @@ class TeamForm extends Component {
             onChange={this.handleChange('address')}
           />
         </FormGroup>
+        <FormGroup controlId="teamForm-vote-duration">
+          <ControlLabel>Sort duration</ControlLabel>
```

Comment:
Could you create and import an SCSS file for this file, then add a class to this label that sets `vertical-align: bottom`? It makes it look better next to the button.